plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -558,7 +558,24 @@ private void cloneFromSCM(PomData pomData, String name, String version, File che
         ScmRepository repository = scmManager.makeScmRepository(pomData.getConnectionUrl());
         CheckOutScmResult result = scmManager.checkOut(repository, new ScmFileSet(checkoutDirectory), new ScmTag(scmTag));
 
-        if (!result.isSuccess()) {
+        if (!result.isSuccess() && config.getAlternativePluginOrganization() != null) {
```

Comment:
IIUC, it will first checkout from the "official" repo and if it fails, then checkout from the alternative one?

Reply:
Yes, this is the idea, try to download from official repo of the plugin specific version and if not possible and you provide alternative organization, the try from there

Reply:
Most probably you want to test a set of plugins (by using a war file as input for example) in that war some of the plugins may come for forks but still majority will still reside in official repos, if you do not use a fallback approach you will need to fork the entire set of plugins to be able to run the PCT for all plugins.

Sure you can do locally using sources, but on CI runs that implies having a special purpose pipeline that checkouts the sources and then stashes them before invoking the PCT which is not trivial to implement in k8s environments for example. This approach is much simpler and integrates with usual PCT pipelines trivially